### ISSUE TITLE
NullPointerException TeadsCrashController

### ISSUE THREAD

**Issue by timrijckaert** [teads/TeadsSDK-android#151]
When upgrading from Teads 4.6.0 to 4.7.0 we get following error

```
2020-05-26 11:30:41.343 23430-23430/be.persgroep.advertising.demo E/AndroidRuntime: FATAL EXCEPTION: main
    Process: be.persgroep.advertising.demo, PID: 23430
    java.lang.NullPointerException: Attempt to read from field 'java.lang.String tv.teads.sdk.android.reporter.core.remote.Collector.a' on a null object reference
        at tv.teads.sdk.android.reporter.core.TeadsCrashController.a(TeadsCrashController.java:1)
        at tv.teads.sdk.android.reporter.TeadsCrashReporter.a(TeadsCrashReporter.java:6)
        at tv.teads.sdk.android.remoteConfig.circuitBreaker.CircuitBreaker.<init>(CircuitBreaker.java:10)
        at tv.teads.sdk.android.engine.EngineFacade.<init>(EngineFacade.java:8)
        at tv.teads.sdk.android.engine.EngineFacadeFactory.a(EngineFacadeFactory.java:1)
        at tv.teads.sdk.android.TeadsAd.<init>(TeadsAd.java:35)
        at tv.teads.sdk.android.CustomAdView.load(CustomAdView.java:13)
        at tv.teads.sdk.android.InReadAdView.load(InReadAdView.java:2)
```

The config is the same in both case.

**Comment by benjaminvolland** [comment]
hello @timrijckaert,
Thank you for your report. We deployed 4.7.0 on Wednesday with a corrupt version, yesterday we corrected it with the correct one.
If you clean your gradle cache and re-synch your dependencies the sdk will work.
To clean up the cache, you can go to Android studio `File> Invalidate caches and restart`. Sometimes this manipulation is not enough, you have to delete it manually (on mac os `Users> .gradle> caches`)

**Comment by timrijckaert** [comment]
Tried it two times now.
It is not working, can you just release a bug fix version instead of releasing it on the same version?

I have this aar in my caches after clearing
https://drive.google.com/file/d/1tpKv9_W1eIr1-N1D2vYlisEpe6s_M3qf/view?usp=sharing

**Comment by timrijckaert** [comment]
I figured out what is causing my issue.
We have an Artifactory/Nexus server in place which is configured to cache dependencies from the upstream Maven repo from Teads.

```
maven {
   url "http://dl.bintray.com/teads/TeadsSDK-android"
}
```

In order to fix it I should either invalidate the cache on the Artifactory too or you guys could simply release a bug fix version.
My preference would be to go with the bug fix version since that is how you should deal with bugs, plus it would work for everyone without manually deleting/invalidating local/remote caches.

Thanks

**Comment by benjaminvolland** [comment]
@timrijckaert We just released the 4.7.1

**Comment by timrijckaert** [comment]
Much thanks for the quick answers and fix.
